### ISSUE TITLE
switching tabs leaves contents of another one visible

### ISSUE THREAD

**Issue by anaderi** [transcranial/atom-transparency#5]
Hi!
as I open the second tab and switch to it, it still displays contents of the 1st one. 
![fff](https://picpig.com/fm56t8ncuy.png)

after I minimize/open window again, everything is fine

OS: mac os X (el capitan) 
Atom: 1.13.0-dev-11f8d9e

**Comment by transcranial** [comment]
@anaderi yeah, you need to start atom with the `--disable-gpu` flag

**Comment by anaderi** [comment]
actually I have such 'ghosts' even with `--disable-gpu`:

![image](https://cloud.githubusercontent.com/assets/1198881/20643821/9a556e40-b433-11e6-886a-ac1bff89e1a2.png)

it might not be visible very well, since I've dimmed colors at stylesheet, but it is clearly there (see the line in the left window starting by `docker_env.sh`.

**Comment by transcranial** [comment]
ah, sorry! I just saw that you're on 1.13.0-dev. I haven't tested that yet. Maybe it has something to do with the removal of the shadow dom? Let me know if you have any ideas.

**Comment by kynetiv** [comment]
Similarly I noticed [hyper.js](https://github.com/zeit/hyper) was leaving ghost remnants of previous text when I tried to just simply adjust the background-color to be more transparent. Then I found [hyper-transparent](https://github.com/codealchemist/hyper-transparent) that fixes it. I wonder if that solution could work here as well.

UPDATE: nevermind, just noticed hyper-transparency has the ghost text problem too. :(

**Comment by iamleet** [comment]
plagued by this too on my mac. -_- - I hot key move the atom window (amethyst) with another and it clears out the ghost, but expect it to return, just a work around.

**Comment by transcranial** [comment]
Yeah, I've had the same issue in macOS, while it works perfectly in ubuntu. I wonder if this would solve it in macOS: https://github.com/electron/electron/issues/8847

**Comment by iamleet** [comment]
Found a fix, works when the window is active. Not really worried about when it isn't tbh. https://github.com/transcranial/atom-transparency/pull/9 Lmk if it helps

**Comment by iamleet** [comment]
@transcranial, dude I love this repo. I found something that works when the atom window is in focus, no burn in on Mac.  https://github.com/iamleet/atom-transparency/tree/issue5_burn_in_text 
Can anyone verify?

**Comment by BioXD** [comment]
Can confirm. The solution posted by iamleet works on 1.25-dev, Sierra 10.13. Just add the following CSS:

`
atom-pane, atom-panel, atom-notification {
  background: rgba(0, 0, 0, 0.5) !important;
  -webkit-backface-visibility: hidden; /* Chrome, Safari, Opera */
  backface-visibility: hidden;
}
`

Then turn off the "background-tips" package and restart atom.

**Comment by iamleet** [comment]
@BioXD  Thanks for confirming, going to submit a pr for the readme

**Comment by ZakharEl** [comment]
Heads up - running linux with compton compositor. Using the --disable-gpu flag as well makes transparency not work whatsoever. Running it with just --enable-transparent-visuals flag works perfectly.
